packages/cypress/integration/demo_reflect.ts

```diff
@@ -0,0 +1,97 @@
+function addCard(column, text) {
+  cy.get(`[data-cy=reflection-column-${column}]`)
+      .find(`[data-cy=phase-item-editor-${column}]`)
+      .find('[data-cy=reflection-editor-wrapper]')
+      .as('column-phase-item-editor')
+    
+  cy.get('@column-phase-item-editor')
+    .type(`${text}`)
+    .should('have.text', `${text}`)
+
+  cy.get('@column-phase-item-editor')
+    .type('{enter}')
+  
+  cy.wait(500)
+
+}
+
+function editCard(column, oldText, newText) {
+  cy.get(`[data-cy=reflection-column-${column}]`)
+    .find(`[data-cy=reflection-stack-${column}]`)
+    .find(`[data-cy=reflection-stack-${column}-card-0]`)
+    .find('[data-cy=reflection-editor-wrapper]')
+    .as('reflection-card-edit')
+
+  cy.get('@reflection-card-edit')
+    .should('have.text', `${oldText}`)
+
+  cy.get('@reflection-card-edit')
+    .type('{selectall}').type('{backspace}').type(`${newText}`)
+    .should('have.text', `${newText}`)
+
+}
+
+function deleteCard(column) {
+      
+  cy.get(`[data-cy=reflection-column-${column}]`)
+    .find(`[data-cy=reflection-stack-${column}]`)
+    .find(`[data-cy=reflection-stack-${column}-card-0]`)
+    .find(`[data-cy=reflection-stack-${column}-card-0-delete]`)
```

Comment:
is it necessary to chain the find commands? what's the advantage over a single `get`?

Reply:
Now that I look at it, it's probably not necessary. Chaining would probably make sense if I used more generic data-cy names. But since the properties here are pretty specific already, there's probably not much point. I guess I just wanted to make extra sure I was getting the right component haha.

Reply:
I'll change it to a single get instead.